user_service/api/views.py

```diff
@@ -0,0 +1,124 @@
+import secrets
+import jose
+from jose import jwt
+from aiohttp import web
+import aioredis
+
+from .db import User
+from .redis import save_pair, del_pair, check_key_redis, set_ttl
+from .utils import get_unique_uuid, generate_jwt
+from .decorators import login_required
+from user_service.config import config
+import user_service.api.errors as errors
+
+
+class LogOut(web.View):
+    @login_required
+    async def get(self):
+        refreshtoken_id = self.request.headers["Authorization"]
+        redis = self.request.app["redis_pool"]
+        is_deleted = await redis.delete(refresh_token_id)
+        if not is_deleted:
+            raise errors.BadLogout
+        else:
+            return web.json_response(
+                data={"status": 204, "message": "Successful logout"}
+            )
+
+
+class Login(web.View):
+    async def post(self):
+        data = await self.request.post()
+        username = data.get("username")
+        password = data.get("pass")
+
+        # check username and password
+        user = await User.verify_user_data(
+            self.request.app["db_pool"], username, password
+        )
+        if user is None:
+            raise errors.BadLoginData
```

Comment:
А тут создать его не надо?
```suggestion
            raise errors.BadLoginData()
```